bip-tx-terminology.mediawiki

```diff
@@ -0,0 +1,286 @@
+
+; Script Arguments
+: [AT] Arguments presented to satisfy the Condition Script, including Signatures, and Public Keys
+
+; Script Tree
+: [AT] The Merkle tree composed from the collection of Leaf Scripts committed to by a P2TR output
+
+; scriptPubKey
+: See _Output Script_
+
+; scriptSig
+: See _Input Script_
+
+; Sequence
+: [TC] The sequence number appearing in each transaction input.  Serialized as the four byte <code>nSequence</code> integer field.  If any input’s sequence number is less than the maximum allowed value, a transaction is non-final.
+
+; Signature
+: [TC] A script argument that proves a key owner’s consent to a transaction. Composed of an r-value, an s-value, and a sighash flag. ECDSA and Schnorr signatures use different serialization schemes for signatures. The length indicator preceding the signature is a serialization artifact that is not part of the signature. If the default `SIGHASH_ALL` is used the sighash flag may be omitted for Schnorr signatures.
+
+; Sighash Flag
+: [TC] Part of the Signature. A one-byte field that indicates which parts of the transaction a signature commits to.
+
+; Taproot Branch
+: [TC] Part of a Control Block that specifies the hashing partners necessary to prove the existence of a Leaf Script in a Script Tree.
+
+; Taproot Internal Key
+: [TC] (aka Inner Key). The untweaked public key before it is tweaked with the Merkle root of the Script Tree to create the Taproot Output Key. Appears as part of a Control Block for a script path P2TR input.
+
+; Taproot Output Key
+: [TC] The public key present in the Witness Program of a P2TR output script composed by tweaking the Inner Key with the Script Tree's Merkle root.
+
+; Tapscript
+: [AT] The variant of the Script language used in P2TR Leaf Scripts (see BIP342).
+
+; Transaction Header
+: [AT] Collective term for the serialization artifacts and transaction fields that appear only once in the transaction serialization regardless of lengths of the Input and Output Lists: Transaction Version, Marker (segwit only), Flag (segwit only), length of Transaction Input List, length of Transaction Output List, Locktime
+
+; Transaction Input
+: [TC] aka Input. An element of the input list of the transaction. Must include an outpoint, an input script, and a sequence. The input script can have various content, corresponding to the UTXO that is being spent. The input script has length 0 for native segwit inputs. In segwit transactions, every input must be accompanied by a Witness Stack. The Witness Stack is not part of the Transaction Input.
+
+; Transaction Input List
+: [AT] The enumeration of all Transaction Inputs of a transaction.
+
+; Transaction Output
+: [TC] Creates a UTXO by specifying an amount and an output script.
+
+; Transaction Output List
+: [AT] The enumeration of all Transaction Outputs of a transaction
+
+; txid
```

Comment:
```suggestion
; Txid
```
for consistency? (also wtxid below)

Reply:
Mh, maybe I’ll rather lowercase everything else

Reply:
Working on making capitalization consistent across the definition list!